src/common/style/base.css

```diff
@@ -49,8 +49,8 @@ q:before, q:after {
 }
 
 a {
-  color: #dd4814;
-  text-decoration: none;
+  color: $cool-grey;
```

Comment:
@earnubs Layout are grayscale wireframes and Vanilla has coloured links, so I'm not so sure about that...